src/Copy/Text.elm

```diff
@@ -243,6 +243,9 @@ t key =
         BackToPartnersLinkText ->
             "Go to all partners"
 
+        SeeOnGoogleMapText ->
+            "See on Google map..."
```

Comment:
Is there a style guide anywhere to provide guidance on what this label should be? The ellipsis looks odd to me.

Reply:
Back in the olden times buttons that opened dialogs would have elipses on them like "save as..." as opposed to "save".

i could put in one of those icon things that [wikipedia uses](https://en.wikipedia.org//w/skins/Vector/resources/common/images/link-external-small-ltr-progressive.svg)

Reply:
Does this open a dialog? That's what's throwing me. This isn't a modal where we ask the user for more information before doing something, it's an action that executes immediately.